README.md

```diff
@@ -15,4 +15,46 @@ The first parameter <PATH OF "auth.whisk.system"> is mandatory and the second pa
 <OPENWHISK_HOME> will be set to the parent directory of the openwhisk-catalog if it is not provided.
 If openwhisk-catalog directory is put in the openwhisk home directory, we are able to omit the
 second parameter. Otherwise, we need to specify the correct OPENWHISK_HOME either as the second
-parameter or the environment variable to make it run.
\ No newline at end of file
+parameter or the environment variable to make it run.
+
+# About the relationship of OpenWhisk and OpenWhisk-Catalog
+
+All the packages under OpenWhisk-Catalog originates from the catalog directory under OpenWhisk. They contain
+the actions to be installed in OpenWhisk. Since OpenWhisk will become the project which only hosts the core
+components, all the packages will be moved into OpenWhisk-Catalog for a better distributed structure.
+
+OpenWhisk-Catalog depends on OpenWhisk installation. All the test cases here depends on the OpenWhisk test
+dependencies as well. For all the test cases, the build.gradle file for tests in OpenWhisk-Catalog will
+add the test dependencies from OpenWhisk.
+
+# How to install OpenWhisk-Catalog
+
+If you have already installed openwhisk:
+Make sure an environment variable <OPENWHISK_HOME> is correctly set for your machine.
+Open a terminal, and go to the directory of openwhisk-catalog/packages. Run the command
+"./installCatalog.sh <OPENWHISK_HOME>/config/keys/auth.whisk.system".
+
+If you have not installed openwhisk:
```

Comment:
We do not have to tell people how to install openwhisk here. The first sentence or paragraph of the section "How to install openwhisk-catalog" (lowercase) should state that installation of openwhisk-catalog assumes the user has installed openwhisk already as a pre-requisite and provide a link to the openwhisk README for instructions.